evmcore/tx_validation_test.go

```diff
@@ -16,154 +16,72 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-func TestValidateTx_BeforeEip2718_RejectsNonLegacyTransactions(t *testing.T) {
-	for name, tx := range getTxsOfAllTypes() {
-		if _, ok := tx.(*types.LegacyTx); ok {
-			continue // Skip legacy transactions
-		}
-		t.Run(name, func(t *testing.T) {
-			err := validateTx(types.NewTx(tx), validationOptions{eip2718: false})
-			require.ErrorIs(t, ErrTxTypeNotSupported, err)
-		})
+// getTestPoolOptions returns a set of options to adjust the validation of transactions
+// considering them as local transactions with a min tip of 1.
+func getTestPoolOptions() poolOptions {
+	return poolOptions{
+		minTip:       big.NewInt(1),
+		isLocal:      true,
+		currentState: nil,
+		locals:       nil,
 	}
 }
 
-// getTestTransactionsOption returns a set of options to adjust the validation of transactions
-// so that it would accept all types of transactions, considering them as local transactions
-// with a min tip of 1, current base fee of 1, and a current max gas of 100_000.
-func getTestTransactionsOption() validationOptions {
-	return validationOptions{
-		eip2718:        true,
-		eip1559:        true,
-		shanghai:       true,
-		eip4844:        true,
-		eip7623:        true,
-		eip7702:        true,
-		currentMaxGas:  100_000,
-		currentBaseFee: big.NewInt(1),
-		minTip:         big.NewInt(1),
-		isLocal:        true,
-		signer:         types.NewPragueSigner(big.NewInt(1)),
+// getTestBlockState returns a test block state database that can be used for
```

Comment:
"database"?

Reply:
no database, just max gas and basefee. comment rephrased.